cmd/minishift/cmd/profile/profile_set.go

```diff
@@ -61,6 +62,28 @@ func runProfile(cmd *cobra.Command, args []string) {
 		}
 	}
 	fmt.Printf("Profile '%s' set as active profile\n", profileName)
+
+	err := removeOcContext(lastActiveProfile)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
```

Comment:
I do not think we should print error here as we are all ready printing it it `removeOcContext`.

Reply:
we only print error in `removeOcContext` if logs are enabled `glog.V(2)`

Reply:
@anjannath That's fine IMO.